### PR TITLE
feat(kubernetes-event-exporter): Enable support for kubeQps/kubeBurst…

### PR DESCRIPTION
**Pull request by DavidHe1127**
…/maxEventAgeSeconds

### Description of the change

Enable settings of 3 params.

### Benefits

Allow settings of these values. They are available in https://github.com/resmoio/kubernetes-event-exporter#troubleshoot-events-discarded-warning.

### Applicable issues

- fixes #

### Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [ ] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [ ] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by carrodher** [comment]
Thanks for reporting this issue. Would you like to contribute by creating a PR to solve the issue? The Bitnami team will be happy to review it and provide feedback. [Here](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md) you can find the contributing guidelines.

In the same way please note it is needed to bump the chart version in the _Chart.yaml_ file

**Comment by DavidHe1127** [comment]
> Thanks for reporting this issue. Would you like to contribute by creating a PR to solve the issue? The Bitnami team will be happy to review it and provide feedback. [Here](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md) you can find the contributing guidelines.
> 
> In the same way please note it is needed to bump the chart version in the _Chart.yaml_ file

Thanks! I've bumped the minor version. Can you have another look please?

**Comment by carrodher** [comment]
Please, note it is needed to sign the commits. Your signature certifies that you wrote the patch or otherwise have the right to contribute the material. You can find more info about this topic, in [the “Sign your work” section from the contributing guidelines](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work).

File: bitnami/kubernetes-event-exporter/Chart.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ name: kubernetes-event-exporter
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/kubernetes-event-exporter
   - https://github.com/resmoio/kubernetes-event-exporter
-version: 2.3.1
+version: 2.4.1
```
Comment (carrodher) [inline bitnami/kubernetes-event-exporter/Chart.yaml:29]:
```suggestion
version: 2.4.0
```

**Comment by DavidHe1127** [comment]
@carrodher version fixed now. Sorry my bad for the wrong verison

**Comment by carrodher** [comment]
It still needed to sign the commits, see detailed instructions at https://github.com/bitnami/charts/pull/16376/checks?check_run_id=13303538694

**Comment by DavidHe1127** [comment]
@carrodher Sorry about all my ignorance. I've now re-committed my changes with right sign-off message. Now all checks pass. Can you merge it?

**Comment by DavidHe1127** [comment]
@carrodher I wasn't able to resolve conflicts through console which the page become unresponsive. Tried from terminal but got the error below

```
To github.com:DavidHe1127/charts.git
 ! [remote rejected]     enable-support-qps-burst-maxeventageseconds -> enable-support-qps-burst-maxeventageseconds (failure)
error: failed to push some refs to 'github.com:DavidHe1127/charts.git'
```

any idea?

**Comment by DavidHe1127** [comment]
@carrodher I wasn't able to resolve conflicts through console which the page become unresponsive. Tried from terminal but got the error below

```
To github.com:DavidHe1127/charts.git
 ! [remote rejected]     enable-support-qps-burst-maxeventageseconds -> enable-support-qps-burst-maxeventageseconds (failure)
error: failed to push some refs to 'github.com:DavidHe1127/charts.git'
```

any idea?

**Comment by DavidHe1127** [comment]
@carrodher I wasn't able to resolve conflicts through console which the page become unresponsive. Tried from terminal but got the error below

```
To github.com:DavidHe1127/charts.git
 ! [remote rejected]     enable-support-qps-burst-maxeventageseconds -> enable-support-qps-burst-maxeventageseconds (failure)
error: failed to push some refs to 'github.com:DavidHe1127/charts.git'
```

any idea?

**Comment by DavidHe1127** [comment]
@carrodher I wasn't able to resolve conflicts through console which the page become unresponsive. Tried from terminal but got the error below

```
To github.com:DavidHe1127/charts.git
 ! [remote rejected]     enable-support-qps-burst-maxeventageseconds -> enable-support-qps-burst-maxeventageseconds (failure)
error: failed to push some refs to 'github.com:DavidHe1127/charts.git'
```

any idea?

**Comment by DavidHe1127** [comment]
Not sure if these heaps of changes relates to the way I've checkout the repo which is sparse checkout to ensure I only checkout the target subdir. It seems it's not easy to fix this PR. I will reopen a new one with full checkout.

**Comment by DavidHe1127** [comment]
reopened  #16544 @carrodher @corico44 Thanks

**Pull request closed without merging**